src/USER-MISC/fix_bond_react.h

```diff
@@ -26,6 +26,8 @@ FixStyle(bond/react,FixBondReact)
 
 #include "fix.h"
 
+#define MAXLINE 256
```

Comment:
sorry for being such a nitpicker, but putting this kind of define here is a bad idea, as it will pollute namespace and potentially cause conflicts in places where all style defining header files need to be included. You can have the same effect in a cleaner fashion by adding `enum {MAXLINE=256};` _inside_ the class definition, which will place it into the class's namespace.

Reply:
thanks, that makes sense